canonical/ubuntu.com
# Add CVEs about page

## Done

- Continuation from this PR https://github.com/canonical/ubuntu.com/pull/14019 due to incorrect branching
- This branch extends from the feature branch
- Added CVEs about page
- Refactored `/security/cve` to `/security/cves` and run djlint formatting on all affected files

## QA

- Go to https://ubuntu-com-14042.demos.haus//security/cves/about
- Check that design and details adheres to Figma and copy docs
- Check that side nav works
  - It should jump to heading on select
  - When scrolling through headings, the side nav links will update automatically


## Issue / Card

Fixes [WD-11852](https://warthogs.atlassian.net/browse/WD-11852)


## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)


[WD-11852]: https://warthogs.atlassian.net/browse/WD-11852?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
